scripts/local/apply-all-migrations.sh

```
@@ -1,6 +1,31 @@
+ENV_FILE=".env"
+case "$TARGET_ENV" in
+  "dev"|"")
+    ENV_FILE=".env"
+    check_docker
+    ;;
+  "staging" | "production")
+    ENV_FILE=".env.$TARGET_ENV"
+    ;;
+  *)
+    # Invalid argument
+    printf "${DANGER}Invalid argument: '$TARGET_ENV'. Usage: $0 [-psh] [dev|staging|production]${NC}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)